llvm/lib/Transforms/Instrumentation/BoundsChecking.cpp

```diff
@@ -108,14 +108,17 @@ static Value *getBoundsCheckCond(Value *Ptr, Value *InstVal,
   return Or;
 }
 
-static CallInst *InsertTrap(BuilderTy &IRB, bool DebugTrapBB) {
+static CallInst *InsertTrap(BuilderTy &IRB, bool DebugTrapBB,
```

Comment:
Changes in this file look unrelated, can you please extract into a separate PR?

Reply:
Done (https://github.com/llvm/llvm-project/pull/124613)